middlewares/writeaction.go

```diff
@@ -0,0 +1,32 @@
+package middlewares
+
+import (
+	"errors"
+	"fmt"
+	"github.com/avigailbr/book_service/datastore"
+	"github.com/gin-gonic/gin"
+	"net/http"
+)
+
+var counter float64
```

Comment:
Is it being used? if so - where? couldn't find it

[I don't like package scoped variables](https://dave.cheney.net/2017/06/11/go-without-package-scoped-variables), unless there's a really good reason to hold them

Reply:
Nope, I forgot to delete it :|